experiment/run_experiment.py

```diff
@@ -448,28 +451,26 @@ def main():
                                required=False,
                                default=None,
                                choices=all_fuzzers)
-    fuzzers_group.add_argument('-cf',
-                               '--changed-fuzzers',
-                               help=('Use fuzzers that have changed since the '
-                                     'last experiment. The last experiment is '
-                                     'determined by the database your '
-                                     'experiment uses, not necessarily the '
-                                     'fuzzbench service'),
-                               action='store_true',
-                               required=False)
+    fuzzers_group.add_argument('-s',
+                               '--no-seeds',
+                               help='Should trials be conducted without seed corpora.',
+                               required=False,
+                               default=False,
+                               action='store_true')
+    fuzzers_group.add_argument('-d',
+                               '--no-dictionaries',
+                               help='Should trials be conducted without dictionaries.',
+                               required=False,
+                               default=False,
+                               action='store_true')
 
     args = parser.parse_args()
-
-    if args.changed_fuzzers:
-        fuzzers = experiment_changes.get_fuzzers_changed_since_last()
-        if not fuzzers:
-            logs.error('No fuzzers changed since last experiment. Exiting.')
-            return 1
-    else:
-        fuzzers = args.fuzzers or all_fuzzers
+    fuzzers = args.fuzzers or all_fuzzers
```

Comment:
this needs to be reverted

Reply:
I didn't include this in the description, but we don't need this code anymore. I can remove here or in another PR up to you.

Reply:
ok, can you remove the changed_fuzzers argument as well.

Reply:
Which argument? I don't see it.

Reply:
ah i didnt see it properly, you did remove -changed-fuzzers